doc/widgets/images/Bag-of-Words-tags.txt

```diff
@@ -0,0 +1,3 @@
+0	253	38
```

Comment:
This is generated with stamper.py and it locates number tags on the picture. We had the same discussion with Kernc some time ago and decided to keep these file in the PR. Useful if you simply need to alter the tags a bit and not do them all over again.